gwvolman/tasks.py

```diff
@@ -294,15 +294,22 @@ def publish(item_ids,
 
 @girder_job(title='Create ad-hoc Tale')
 @app.task(bind=True)
-def create_adhoc_tale(self, imageId: str, dataId: List[str]):
+def create_adhoc_tale(self, imageId: str, dataId: List[str], spawn: bool):
```

Comment:
Very minor, but the idea that this is an "Ad-hoc" tale seems to only apply for the dataset analysis case.  If importing a tale in the future, then it isn't really ad-hoc.  Will this same method be used in that case?